src/renderer/components/+workloads-pods/pod-details-secrets.tsx

```diff
@@ -13,33 +13,49 @@ interface Props {
 
 @observer
 export class PodDetailsSecrets extends Component<Props> {
-  @observable secrets: Secret[] = [];
+  @observable secrets: Map<string, Secret> = observable.map<string, Secret>();
```

Comment:
```suggestion
  @observable secrets = observable.map<string, Secret>();
```

_Reason:_ `observable.map` has more api methods over standard `Map`, e.g. `map.toJS()`, `map.replace()`, etc.